pyslurm/pyslurm.pyx

```diff
@@ -2160,7 +2160,7 @@ cdef class job:
             Job_dict[u'start_time'] = self._record.start_time
 
             if self._record.state_desc:
-                Job_dict[u'state_reason'] = self._record.state_desc.replace(" ", "_")
+                Job_dict[u'state_reason'] = self._record.state_desc.decode().replace(" ", "_")
```

Comment:
In theory yes. In practice I'd consider it a bug, if slurm did supply non-ASCII characters in a job field. It's up to you.